sources/web/datalab/polymer/components/datalab-files/datalab-files.ts

```diff
@@ -233,6 +234,8 @@ class FilesElement extends Polymer.Element {
         }
       })
       .then(() => this._fetching = false);
```

Comment:
We still need `_fetching` for displaying the progress bar, and it's also used in other places, where using the promise directly won't be very clear. I'll keep using the flag for now.